<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_loan</class>
 <widget class="QWidget" name="search_loan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>899</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>899</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search Loan</string>
  </property>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>14</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>42</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpf_field_input">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>14</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text_input">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>18</y>
     <width>44</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>493</y>
     <width>230</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_text">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>68</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_output">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>70</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="installment_text">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>51</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Installments</string>
   </property>
  </widget>
  <widget class="QLabel" name="installment_paid_output">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>49</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="installment_paid_text">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Installment paid</string>
   </property>
  </widget>
  <widget class="QLabel" name="fees_output">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>81</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fees_text">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>84</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fees</string>
   </property>
  </widget>
  <widget class="QLabel" name="installments_output">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>49</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amount_output">
   <property name="geometry">
    <rect>
     <x>741</x>
     <y>14</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amount_text">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>14</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QComboBox" name="loan_list_box">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>100</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="loan_list_text">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>102</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loans</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_output">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>101</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text_output">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_text">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>11</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_output">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>13</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>180</y>
     <width>871</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
  </widget>
  <widget class="QLabel" name="amount_month_output">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>83</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amount_month_text">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>83</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount per month</string>
   </property>
  </widget>
  <widget class="QLabel" name="payday_text">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>114</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Payday</string>
   </property>
  </widget>
  <widget class="QLabel" name="payday_output">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>111</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="loan_quantity_output">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>113</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="loan_quantity_text">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>114</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loan quantity</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cpf_field_input</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
